modules/apps/transfer/types/transfer_authorization.go

```diff
@@ -3,7 +3,6 @@ package types
 import (
 	"math/big"
 
-	errorsmod "cosmossdk.io/errors"
```

Comment:
I reverted this change that came in with this [backport PR](https://github.com/cosmos/ibc-go/pull/3580/files), since using the errors package was introduced in v7.1.x